### PR TITLE
test-case: Add auto pause-resume test with playback & capture

### PR DESCRIPTION
**Pull request by stripes416**
pause-resume on simultaneous playback and capture pipelines

Signed-off-by: Kelly Ledford <kelly.ledford@intel.com>

### CODE REVIEW

File: test-case/check-pause-resume-simultaneous-playback-capture.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+if [ "$pcm_p" = "" ]||[ "$pcm_c" = "" ];
+then
+    dloge "No playback or capture PCM is specified. Skip the pause-resume test"
+    exit 2
+fi
+
+[[ ${OPT_VALUE_lst['s']} -eq 1 ]] && func_lib_start_log_collect
+
+channel=2
+rate=4800
```
Comment (fredoh9) [inline test-case/check-pause-resume-simultaneous-playback-capture.sh]:
```suggestion
rate=48000
```

**Comment by xiulipan** [comment]
close as #67 #68 #69 are same thing,

**Comment by stripes416** [comment]
@xiulipan I dont think this should be closed. I responded to your reasoning on the framework PR.

**Comment by stripes416** [comment]
closing --- due to #76

**Pull request closed without merging**